evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -40,9 +40,15 @@ def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, ma
                       "max_depth": max_depth,
                       "num_leaves": num_leaves,
                       "min_child_samples": min_child_samples,
-                      "n_jobs": n_jobs}
+                      "n_jobs": n_jobs,
+                      "bagging_freq": bagging_freq,
+                      "bagging_fraction": bagging_fraction}
         parameters.update(kwargs)
 
+        # if the boosting type is random forest, we want to change the bagging_freq to 1 so that we avoid errors
+        if boosting_type == "rf" and not bagging_freq:
+            parameters.update({'bagging_freq': 1})
```

Comment:
I'm updating the bagging frequency here, but I think it would be nice to update the user with this. Can I add logging/printing to the component itself?

Reply:
@bchen1116 see my comment above. Perhaps we should prevent automl from recommending this case when `boosting_type="rf"` and `bagging_freq=0`. And then, for direct usage of the component, we could either raise an error when this combination of parameters is used, or let lightgbm raise the error. How ugly is the error?

Reply:
Resolved by comment above.